src/WidenImage.css

```diff
@@ -0,0 +1,61 @@
+
```

Comment:
:+1: Thanks for your thoughts on this. Let's keep on thinking but it's nothing to worry about now.